### PR TITLE
The next blog post

### PR DESCRIPTION
**Pull request by mirkoboehm**
Let's get this out today.

### CODE REVIEW

File: content/blog/qmstr-newsletter-8.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+<!--more-->
+
+# Reporting API refactored, implemented in HTML reporter
+
+The elemental workflow from construction to analysis to reporting has
+become more and more defined throughout the development of version
+0.1. In sprint #4 we defined the analysis API, which was a dependency
+for the reporting API. Both are defined as gRPC interfaces to the
+master. This makes module-side implementations in different
+programming languages a possibility, since modules always are run as
```
Comment (hemarkus) [inline content/blog/qmstr-newsletter-8.md]: are run => run or are running

File: content/blog/qmstr-newsletter-8.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+categories = ["qmstr"]
+banner = "img/blog/qmstr-newsletter-8-title.jpg"
++++
+
+Spring is coming, and so is Quartermaster v0.1! One more sprint to get
+there. Sprint 5 saw the implementation of initial support for Gradle
+based Java projects, a finished definition of the reporting API, the
+introduction of "easy mode" (see below), improved author detection and
+more steps towards an automated HTML reporter. Sprint 5 was the last
+roud of new features before Quartermaster v0.1 will be wrapped up for
```
Comment (Katajau) [inline content/blog/qmstr-newsletter-8.md]: round

File: content/blog/qmstr-newsletter-8.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+become more and more defined throughout the development of version
+0.1. In sprint #4 we defined the analysis API, which was a dependency
+for the reporting API. Both are defined as gRPC interfaces to the
+master. This makes module-side implementations in different
+programming languages a possibility, since modules always are run as
+separate processes by the master.
+
+Keeping the administrative effort to run a build in Quartermaster low
+and simple is a key goal for us. With the flexibility of the phased
+workflow toolchain comes some setup effort to configure how the
```
Comment (Katajau) [inline content/blog/qmstr-newsletter-8.md:33]: ...comes with some setup....

File: content/blog/qmstr-newsletter-8.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+arsenal. It may learn how to set up the environment for a Python based
+analysis module, for example. Easy mode is still work in progress, and
+comes with a big fat warning when it starts. Feel free to try it.
+
+## Plans for the upcoming (final v.01) sprint
+
+There is one more sprint before the first Quartermaster milestone is
+completed. The Quartermaster project is organised into milestones that
+always span one quarter. Within each milestone, there are six sprints
+that last 2 week each. At the end of every sprint there is a public
```
Comment (Katajau) [inline content/blog/qmstr-newsletter-8.md]: ....last 2 weeks each.

File: content/blog/qmstr-newsletter-8.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+demo on Wednesday at 4pm CET (or CEST). At the end of the milestone,
+there is always a requirements workshop for the coming milestone that
+is open to the whole community. If you are interested in joining, get
+in touch through one the open communication channels with the project:
```
Comment (Katajau) [inline content/blog/qmstr-newsletter-8.md]: ....one of the....

File: content/blog/qmstr-newsletter-8.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+Chain](http://ossg.bcs.org/blog/2018/02/15/yanking-the-chain-open-source-software-compliance-in-the-supply-chain-london-22-3-2018/)
+on March 22 in London. People seem to have enjoyed it. There are
+Quartermaster bubbles on screen!
+
+{{< tweet 976858824006135808 >}}
+
+## How to participate
+
+The milestone 2 requirements workshop will be held on April 11, 2018
+from 10am to 5pm, at the Endocode office on Berlin. Remember to
```
Comment (Katajau) [inline content/blog/qmstr-newsletter-8.md]: ....office in Berlin.

**Review by Katajau — COMMENTED** [review]
Looks great, just a few minor remarks.

File: content/blog/qmstr-newsletter-8.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+Keeping the administrative effort to run a build in Quartermaster low
+and simple is a key goal for us. With the flexibility of the phased
+workflow toolchain comes some setup effort to configure how the
+modules (running in separate processes) communicate with the
+master. This may sound simple enough for now, however to scale larger
+Quartermaster deployments, it may be necessary for the modules to run
+distributed over different machines. Or, depending on the build
+environment, the whole Quartermaster deployment could run in a
+Kubernetes cluster and execute the modules in nodes. Quartermaster is
```
Comment (hemarkus) [inline content/blog/qmstr-newsletter-8.md]: Prepare but not yet ready to do that.

**Review by hemarkus — CHANGES_REQUESTED** [review]
Minor grammar fix needed

File: content/blog/qmstr-newsletter-8.md (hunk 3 above)
Comment (mirkoboehm) [reply content/blog/qmstr-newsletter-8.md:33]: "With the flexibility of the phased workflow toolchain comes some setup effort" I think is correct.

**Pull request merged**